packages/utils/src/content-type.ts

```diff
@@ -452,6 +452,7 @@ const getType = (mediaType: string) => {
     switch (mediaType) {
         case 'application/javascript':
         case 'text/javascript':
+        case 'text/jsx':
             return 'script';
         case 'text/css':
```

Comment:
I added `text/jsx` to map to `script` above so we get `fetch::end::script` events for `*.jsx` files. However we don't currently do something similar for `*.sass` and `*.less` files to map them to `fetch::end::css` (probably would have if it had been `fetch::end::style`) so they map to `fetch::end::unknown`.

I feel we probably should make these consistent one way or the other. Thoughts?

Reply:
I'm torn apart with this.
On one side I think that mapping to `unknown` is a bit obscure for developers. On the other side we will have to add a bunch of things to `utils` and thus release everything again.

If we decide to add more mappings I think we should try add as many in one go as possible, maybe even some that don't have a parser yet.

Reply:
Yeah, I'm having the same internal conflict. 

Personally I'd almost like to make the events themselves flexible so we automatically have `fetch::end::jsx`, etc. based on content type, but I wouldn't be able to type it correctly. I think I'm okay keeping just JSX special as we intend to parse it with our JS parser to extract normal JS content contained within (whereas LESS and SASS require fully dedicated parsers).

Another thought is having a generic `fetch::end::text` event for all `text/*` content types that don't map to something more specific (slightly better than `unknown`).

Reply:
Given we don't want to make a habit of this, I decided to revert `text/jsx` to `unknown` for now. Plus that was the only reason left this PR was touching the `utils` package and it didn't seem worth it.